src/cider/tasks.clj

```diff
@@ -0,0 +1,17 @@
+(ns cider.tasks
+  (:require [boot.core :refer [deftask]]
+            [boot.repl :as repl]
+            [boot.util :as util]))
+
+(deftask add-middleware
+  "CIDER middleware injection task
+
+  This task allows to inject middleware in boot.repl/*default-middleware*.
+  Just pass them as -m|-middleware. The input is a string but will be converted
+  to symbol."
+  [m middleware MW #{str} "String of the middleware to inject"]
```

Comment:
Boot tasks usually take a symbol there to identify the argument: https://github.com/boot-clj/boot/wiki/Task-Options-DSL